sphinx_gallery/gen_gallery.py

```diff
@@ -294,7 +294,8 @@ def setup(app):
 
     app.add_stylesheet('gallery.css')
 
-    if 'sphinx.ext.autodoc' in app._extensions:
+    extensions_attr = '_extensions' if hasattr(app, '_extensions') else 'extensions'
```

Comment:
Hmm, well indeed the alternative is not spotless as ``'extensions'`` doesn't mean anything as a string. Haven't tested that nesting works or not as I have to run now:
```
if 'sphinx.ext.autodoc' in getattr(app, '_extensions', getattr(app, 'extensions')):
```